.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 0 * * 1'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  static:
+    uses: dvsa/.github/.github/workflows/php-library-static.yml@v3.2.3
+
+  test:
+    uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
+    with:
+      php-versions: "[\"7.4\", \"8.0\", \"8.1\"]"
```

```suggestion
      php-versions: "[\"8.0\", \"8.1\"]"
```
v7 not needed here. `composer.json` for this repo uses at least v8.0